src/layouts/BaseLayout.astro

```diff
@@ -21,6 +24,12 @@ const { title, description } = Astro.props;
 		<meta name="viewport" content="width=device-width" />
 		<meta name="generator" content={Astro.generator} />
 		<title>{title ? `${title} | Interledger Foundation` : 'Interledger Foundation'}</title>
+
+		<meta property="og:title" content={title ? `${title} | Interledger Foundation` : 'Interledger Foundation'} />
```

Comment:
Exposing `ogType` to the next layer down because I want individual blog posts to have type of `article`. Similar concept for the og-image.